tests/unit/test_proxy.py

```diff
@@ -37,16 +39,18 @@ def test_proxy_from_config_non_arg(self):
 
     def test_proxy_from_config_kubernetes(self):
         """Tests that a Kubernetes proxy is correctly created from config."""
-        proxy = proxy_from_config({
-            "type": "kubernetes-port-forward",
-            "config": {
-                "api-host": "https://localhost:8456",
-                "namespace": "controller-python-test",
-                "remote-port": "1234",
-                "service": "controller",
-                "service-account-token": "==AA",
-                "ca-cert": "==AA",
-            },
-        })
+        proxy = proxy_from_config(
```

Comment:
Please prefer pre-commit over other tools.

This change seems to be the result of `ruff format`, while the original was the result of `ruff format --preview`.